web/packages/teleport/src/Integrations/status/AwsOidc/Cards/ConsoleCard.tsx

```diff
@@ -100,46 +71,25 @@ function EnrolledCard({
         <Flex gap={3} my={3}>
           <Box>
             <Text fontWeight="500" fontSize={9} mb={2} css={{ lineHeight: 1 }}>
-              {profiles}
+              {syncedProfiles || 0}
```

Comment:
`syncedProfiles` is a number, so the only falsey value is `0` anyway. can you explain this a bit more?

Reply:
There was an issue early on where the stats endpoint wasn't returning the top level field, the story `No data` captures the UX if so.